frontend/src/static/js/main.js

```diff
@@ -1,3 +1,21 @@
+const Util = {
```

Comment:
Hmm, I'm not so sure about this way of creating a util method is the best way. @ang-zeyu may I get your opinion on this?